spotlight-client/src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
Nit: Let's make sure we've got license text at the top of each file in the app. We didn't do that in Covid-19 Dashboard but that was simply because I didn't bother to add that extra bit of friction during that time. But we should do it for every file we generate going forward that supports comments.